src/headers.js

```diff
@@ -130,47 +131,20 @@ class Headers {
   }
 
   forEach (callback, thisArg) {
-    this[kHeaders] = sort(this[kHeaders])
+    this[kHeaders] = sort1d(this[kHeaders])
     for (let i = 0; i < this[kHeaders].length; i += 2) {
       callback.call(thisArg, this[kHeaders][i + 1], this[kHeaders][i], this)
     }
   }
 
   * [Symbol.iterator] () {
-    this[kHeaders] = sort(this[kHeaders])
+    this[kHeaders] = sort1d(this[kHeaders])
```

Comment:
```suggestion
    sort1d(this[kHeaders])
```

remove the reassignment since the algorithm mutates in place